airbyte-webapp/package.json

```diff
@@ -16,10 +16,11 @@
     "format": "prettier --write 'src/**/*.{ts,tsx}'",
     "storybook": "start-storybook -p 9009 --quiet",
     "build:storybook": "build-storybook -o 'build/storybook'",
-    "lint": "eslint --ext js,ts,tsx src",
+    "lint": "eslint --ext js,ts,tsx src && stylelint \"**/*.{css,scss}\"",
     "license-check": "node ./scripts/license-check.js",
     "generate-client": "orval",
-    "validate-links": "ts-node --skip-project ./scripts/validate-links.ts"
+    "validate-links": "ts-node --skip-project ./scripts/validate-links.ts",
+    "stylelint-check": "stylelint-config-prettier-scss-check"
```

Comment:
Not sure if we should add this to a pipeline somewhere, or just ignore.

Reply:
We should def run that on CI, since it's otherwise not really linting anything. For that we should add a new task to our build.gradle file (you can see similar `type: NpmTask` in there), and make the gradle `check` step depend on this (line 108 you can just add a new task that should run for checks).

We should make also sure that the task running on CI fails if it would restyle any SCSS file, i.e. make git dirty by running. Ideally we see if any of the `stylelint` commands actually has a flag to automatically fail if it needs to touch a SCSS file (and separate this here into two tasks: one that does reformat for running locally, and one for CI that fails when it would reformat).